tools/toolchains/bazel4_local_config_cc/armeabi_cc_toolchain_config.bzl

```diff
@@ -0,0 +1,82 @@
+# Copyright 2019 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I'll treat these files as "copy + pasted" and not review in depth, let me know if that makes sense or not. In case you made any changes I'd appreciate if you could point me to the relevant bits

Reply:
There's 1-2 code line changed - you can grep `bazel4.` in the PR.